### PR TITLE
Update regional run steps

### PR DESCRIPTION
**Pull request by devarajun**
small changes/fixes for a regional run

### CODE REVIEW

File: content/run.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ There are dedicated tools and workflow for running single-cell simulations over
 Follow the below procedure to run CTSM over a specific region of interest for a specific resolution. 
 
 ### Domain and surface data
-To run over the Scandinavia region (latitude 48N to 81N, longitude 4E to 42E) at 0.5 degree resolution you first need to produce the domain and surface data files for the region using the python script `subset_data.py` available under the CTSM tools directory `CTSM_ROOT/tools/contrib/subset_data.py`. The python script file can be found [here](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data.py). 
+To run over the Scandinavia region (latitude 48N to 81N, longitude 4E to 42E) at 0.5 degree resolution you first need to produce the domain and surface data files for the region using the python script `subset_data.py` available under the CTSM tools directory `CTSM_ROOT/tools/site_and_regional/subset_data.py`. The python script file can be found [here](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data.py).
```
Comment (ziu1986) [inline content/run.md:146]: Now we have a duplicate - the last two sentences point to the same. I would suggest to merge them and either mention the subdirectory or point to the github page. I tend to the second option, as it is easier to check the validity of a link.

File: content/run.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ input argument variables:
     
 First, you need to change the global input data directory variable inside the script (standard global CTSM dataset) as below:
 
-    dir_inputdata='/cluster/shared/noresm/inputdata/'
+    dir_inputdata='/cluster/shared/noresm/inputdata/'  (somelines after region.create-tag())
```
Comment (ziu1986) [inline content/run.md:155]: Why add this? You can easily look for the dir_inputdata variable in the code.

File: content/run.md [hunk 3]
```diff
@@ -161,12 +161,12 @@ Set the environment variable for the directory as below:
 
 ```{discussion} Choose the global surface data file corresponding to the resolution! 
 For example: for 0.5x0.5 resolution, choose `surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc` (under shared noresm folder)
-and add this file location in the python script `subset_data.py`
+and add this file location in the python script `subset_data.py`. Make sure you have created a conda environment for python including xarray, netcdf4 installed or loaded on FRAM or SAGA.
```
Comment (ziu1986) [inline content/run.md:164]: This doesn't belong here! Should go to line 167.

**Review by ziu1986 — CHANGES_REQUESTED** [review]
Few minor changes before this can be accepted. Thank you for updating the documentation!

File: content/run.md (hunk 1 above)
Comment (devarajun) [reply content/run.md:146]: I think its fine to have that additional information. Link provides to have a quick glance of the script. if its just a link then users may think of downloading it.

File: content/run.md (hunk 2 above)
Comment (devarajun) [reply content/run.md:155]: It seems this is necessary considering Gunnar's experience. There is another place of dir_inputdata for site run, it does not work if you change there.

File: content/run.md (hunk 3 above)
Comment (devarajun) [reply content/run.md:164]: Okay that's a good point. I moved the text there.

File: content/run.md (hunk 1 above)
Comment (ziu1986) [reply content/run.md]: OK. But it is still more work to keep it up-to-date :-).

File: content/run.md [hunk 4]
```diff
@@ -143,7 +143,7 @@ There are dedicated tools and workflow for running single-cell simulations over
 Follow the below procedure to run CTSM over a specific region of interest for a specific resolution. 
 
 ### Domain and surface data
-To run over the Scandinavia region (latitude 48N to 81N, longitude 4E to 42E) at 0.5 degree resolution you first need to produce the domain and surface data files for the region using the python script `subset_data.py` available under the CTSM tools directory `CTSM_ROOT/tools/contrib/subset_data.py`. The python script file can be found [here](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data.py). 
+To run over the Scandinavia region (latitude 48N to 81N, longitude 4E to 42E) at 0.5 degree resolution you first need to produce the domain and surface data files for the region using the python script `subset_data.py` available under the CTSM tools directory `CTSM_ROOT/tools/site_and_regional/subset_data.py`. The python script file can also be found [here](https://github.com/ESCOMP/CTSM/blob/master/tools/site_and_regional/subset_data.py).
```
Comment (ziu1986) [inline content/run.md:146]: OK

File: content/run.md (hunk 2 above)
Comment (ziu1986) [reply content/run.md]: Understood. But could you give perhaps give a line number +/-? The text should be moved and included in line 153 instead. "First, you need to change the variable pointing to the CTSM input data directory on your system. The variable can be found xx lines after the function `region.create-tag()`."

Remark: If you set f.e. a CTSM_INPUTDATA environment variable, the code could be changed to `dir_inputdata = os.environ['CTSM_INPUTDATA']`. Like this, you can easily switch the input directory without changing the script.

File: content/run.md [hunk 5]
```diff
@@ -161,12 +161,12 @@ Set the environment variable for the directory as below:
 
 ```{discussion} Choose the global surface data file corresponding to the resolution! 
 For example: for 0.5x0.5 resolution, choose `surfdata_360x720cru_16pfts_Irrig_CMIP6_simyr2000_c170824.nc` (under shared noresm folder)
-and add this file location in the python script `subset_data.py`
+and add this file location in the python script `subset_data.py`. 
 ```
 
-Run the script as below with the input arguments for Scandinavia region:
+Run the script as below with the input arguments for Scandinavia region (make sure you have created a conda environment for python including xarray, netcdf4 installed):
```
Comment (ziu1986) [inline content/run.md:167]: May remove the parentheseses and conjunct "[...] and make sure [...]". Which flavor of python? 3 or 2?

**Review by ziu1986 — CHANGES_REQUESTED** [review]
Almost there just a few more changes to make it perfect!

**Comment by ka7eh** [comment]
@devarajun what's the state of this PR? Is it still relevant?

**Review by ka7eh — APPROVED** [review]

**Pull request merged**